client/src/document/index.scss

```diff
@@ -419,7 +419,6 @@ pre,
 code,
 .code-example {
   border-radius: var(--elem-radius);
```

Comment:
@caugner  adding the `font-family` here (and therefore to `pre`) seems to fix it. And we should set it anyhow.

Reply:
Fixed in f188f708b.